perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_facility_attribute_def_def_pbsServer.java

```diff
@@ -40,6 +40,11 @@ public void checkAttributeSemantics(PerunSessionImpl perunSession, Facility faci
 		if(!success) throw new WrongAttributeValueException(attribute, "There is no such facility with the same name like this pbsServer");
```

Comment:
I think this whole part is semantic. And the error is that no core attribute "name" for object facility exists in perun.

Reply:
Okay, the check was moved to semantic part and the error message was changed.